scripts/docker_files/docker_file_ci_centos_7_python35/DockerFile

```diff
@@ -16,7 +16,7 @@ RUN ls /usr/src/Python-3.5.4
 RUN sh /usr/src/Python-3.5.4/configure --enable-optimizations --with-lto --enable-shared LDFLAGS="-Wl,-rpath /usr/local/lib"
 RUN make altinstall
 RUN rm Python-3.5.4.tgz
-RUN python3.5 -m pip install 'cx_Freeze==6.1' numpy
+RUN python3.5 -m pip install 'cx_Freeze==6.1' numpy sympy
```

Comment:
Not sure what `cx_Freeze==6.1` so I'm wondering if I should put sympy in a separate line